src/main/kotlin/no/skatteetaten/aurora/boober/model/openshift/ApplicationDeployment.kt

```diff
@@ -22,6 +22,7 @@ data class ApplicationDeploymentSpec(
     val applicationDeploymentId: String,
     val applicationName: String,
     val applicationDeploymentName: String,
+    val databases: List<String>,
```

Comment:
Hvordan skal vi her kunne matche 1 id til et sett med labels fra AuroraConfig? 

Trenger vi ikke enten labels som blir brukt for match eller navn på databasen i AuroraConfig?

Gitt at man har 2 databaser med navn FOO og BAR som begge har auto satt. 
Disse blir provisjonert og får id FOO=1, BAR=2

Da vil vi få en listen med id (1,2). 

For å korrelere dette må vi da til databaseHotellet, men det er kanskje greit?

Reply:
Det er slik vi har designet resten av ApplicationDeployment. Vi har referanser til mer informasjon om deler av konfigurasjonen. Men bør vi ha et felt som sier hvilket api vi skal sende en id til for å få mer informasjon?